test/e2e/common.ts

```diff
@@ -85,17 +85,10 @@ export const launch = async (
   createConductorProcess.stdin.write(LAIR_PASSPHRASE);
   createConductorProcess.stdin.end();
 
-  let conductorDir = "";
   const createConductorPromise = new Promise<void>((resolve) => {
     createConductorProcess.stderr.on("data", (data) => {
       console.error("[hc sandbox] ERROR: ", data.toString());
     });
-    createConductorProcess.stdout.on("data", (data) => {
```

Comment:
Not required anymore since we can run an existing sandbox by index now rather than by specifying the conductor directory.